### ISSUE TITLE
(React Native) How to disable crash reporting but leave user feedback operating

### ISSUE THREAD

**Issue by peacechen** [Instabug/instabug-reactnative#137]
Is there a way to disable just crash reporting, while leaving the user feedback operational? 

One of our apps has been built with another crash reporting tool.  We have opted to leave that in place since it's already implemented and there are features that Instabug doesn't have yet.

**Comment by gianpaj** [comment]
see
https://docs.instabug.com/docs/react-native-popups#section-prompt-options
```js
// Arguments: chat, bug, feedback
Instabug.setPromptOptionsEnabled(false, false, true);
```

**Comment by peacechen** [comment]
@gianpaj 
We want to leave the user feedback operational, but disable the automated crash reporting (when the app crashes).

**Comment by gianpaj** [comment]
oh, i see.

in iOS:
https://docs.instabug.com/docs/ios-crash-reporting#section-disabling-and-enabling
```objectivec
[Instabug setCrashReportingEnabled:NO];
```

On Android:
https://docs.instabug.com/docs/android-crash-reporting#section-disabling-and-enabling
```java
new Instabug.Builder(this, "APP_TOKEN")
    .setCrashReportingState(Feature.State.DISABLED)
    .build();
```

**Comment by peacechen** [comment]
Thanks for the documentation links 👍 

It would be even better if those methods were exposed to the RN JS API so that crash reporting can be set through RN.  That would allow toggling it in the UI.

**Comment by peacechen** [comment]
It appears that iOS crash reporting is disabled when using the RN library:
https://github.com/Instabug/instabug-reactnative/blob/cbf6c2d5d88485627bfbf18b90cd5272f459a019/ios/RNInstabug/InstabugReactBridge.m#L37

**Comment by ZiadAyman** [comment]
hey @peacechen 
Apologies for replying to this a bit late! 

Is everything going good at your end right now?

**Comment by peacechen** [comment]
I spoke too soon.  The method `setCrashReportingState` doesn't appear to be available in the RN library.

```
MainApplication.java:55: error: cannot find symbol
                .setCrashReportingState(Feature.State.DISABLED)
                ^
  symbol:   method setCrashReportingState(State)
  location: class Builder
1 error
:app:compileDebugJavaWithJavac FAILED
```

RN Instabug builds and works without that.
Snippet:
```
    @Override
    protected List<ReactPackage> getPackages() {
      return Arrays.<ReactPackage>asList(
          new MainReactPackage(),
            new RNInstabugReactnativePackage.Builder(instabugKey, MainApplication.this)
            	.setCrashReportingState(Feature.State.DISABLED)
							.setInvocationEvent("shake")
							.setPrimaryColor("#1D82DC")
							.setFloatingEdge("left")
							.setFloatingButtonOffsetFromTop(250)
							.build(),
...
```

**Comment by ZiadAyman** [comment]
@peacechen RN Crash reporting is not supported yet and there is no JS API to disable it for now. We disabled crash reporting in RN from the native SDK so its by default disabled in RN. 

Hopefully once crash reporting is supported on RN which is very soon, there will be a disable and enable JS API for it :)

Let me know if any further assist is needed here!

**Comment by SnidelyWhiplash** [comment]
@ZiadAyman Is there an ETA on this? It seems a little less-than-ideal to have 50% of the product offering missing from the RN integration, especially for paying customers. Would love to hear if this is close to getting completed.

Also worth noting that the README.md mentions crash reporting explicitly. Cheers.

**Comment by HeshamMegid** [comment]
@SnidelyWhiplash JS crash reporting is available starting from version 2.11. Please check the docs at https://docs.instabug.com/docs/react-native-crash-reporting
